server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/ConsistencyTasksIntegrationTest.java

```diff
@@ -394,32 +468,30 @@ void recomputeMailboxCountersShouldSolveNotingWhenNoInconsistencies(GuiceJamesSe
 
         with()
             .basePath(TasksRoutes.BASE)
-            .get(taskId + "/await");
+        .get(taskId + "/await");
 
-        assertThat(probe.retrieveCounters(inbox).getCount()).isEqualTo(1);
```

Comment:
Is probe.retrieveCounters used in over places?

If not, we should delete that code...